packages/components/pie-cookie-banner/custom-elements-manifest.config.mjs

```diff
@@ -0,0 +1,15 @@
+import { moduleFileExtensionsPlugin } from 'cem-plugin-module-file-extensions';
```

Comment:
if the choice is move the CEM config to each component for this PR vs move it once to the components config, then that's preferable in my opinion and we should just read from that package

Reply:
Yep, agreed :)